pkg/collector/client.go

```diff
@@ -32,46 +60,104 @@ func (c *clientCollector) Collect(ch chan<- prometheus.Metric) {
 		return
 	}
 
-	ch <- prometheus.MustNewConstMetric(c.omadaClientConnectedTotal, prometheus.GaugeValue, float64(len(clients)),
-		site, client.SiteId)
+	totals := map[string]int{}
 
 	for _, item := range clients {
 		vlanId := fmt.Sprintf("%.0f", item.VlanId)
 		port := fmt.Sprintf("%.0f", item.Port)
+		connectionModeFmt := ""
 		if item.Wireless {
 			wifiMode := fmt.Sprintf("%.0f", item.WifiMode)
```

Comment:
Maybe we should change this to a string and set the value using `FormatWifiMode(int(item.WifiMode))`? The current value isn't very useful an it saves us having labels for both `wifi_mode` and  `wifi_mode_fmt` (I'm fine with the breaking change, I can add it to the release notes)

Reply:
See latest change, removed ` wifi_mode_fmt`